blueprint/client-devices.md

```diff
@@ -1198,20 +1215,20 @@ Securing iOS devices Design Decisions for all agencies and implementation types.
 Decision Point | Design Decision | Justification
 --- | --- | ---
 Mobile Device Management | Mobile devices will be managed using Intune | Leveraging the capabilities already available in the licensing agreement. Intune and Apple Business Manager will be adopted to manage mobile devices.
-Security policies and hardening requirements | Security policies will be enforced on all mobile devices managed by the Agency | Security policies will be configured in line with the ACSC Security Configuration Guide – Apple iOS 12 Devices.
-Device Features | Disabled | AirDrop, Bluetooth pairing, AirPrint
+Security policies and hardening requirements | Security policies will be enforced on all mobile devices managed by the Agency | Security policies will be configured in line with the ACSC Security Configuration Guide – Apple iOS 14 Devices.
+Device Features | Configured | Device features configured in line with ACSC guidance. 
 
 Mobile Device Management Configuration applicable to all agencies and implementation types
 
 Configuration | Value | Description
 --- | --- | ---
 **Security** | | |
 Supervised Mode | Enable | Agency iPhones are managed devices. This is in line with the ACSC iOS Secure Configuration Hardening guide for PROTECTED devices.
-Device passcode | Device passcode of 14 characters or above | iOS devices, by default, is encrypted once a passcode is provided to the device. Configured in line with ISM requirements on password length.
+Device passcode | Device passcode of 14 characters or above. Alphanumic in nature and must contain a minimum of 1 special character | iOS devices, by default, is encrypted once a passcode is provided to the device. Configured in line with ISM requirements on password length.
 Biometrics | Disable | This is in line with the ACSC iOS Secure Configuration Hardening guide for PROTECTED devices.
 Mobile Device Management | Enable | Mobile Device Management provides the Agency better auditing tools on the device. In line with ACSC iOS Secure Configuration Hardening guide for PROTECTED devices
 Maximum Auto-Lock  | 2 minutes | Auto Lock will lock a device if it is inactive for specified time
-Virtual Private Network (VPN)  | Not Configured | VPN will not be setup as part of the iOS device. Per-app VPN will be set up to secure communication between the device and the Agencies data
+Virtual Private Network (VPN)  | Configured | Per-app VPN will be set up to secure communication between the device and the Agencies data. This is in line with ACSC iOS Secure Configuration Hardening guide
```

Comment:
For clarity, has this become a must in this iteration of the iOS hardening guidance?

Reply:
The latest hardening guidance has a VPN as a Required item


![image](https://user-images.githubusercontent.com/81546609/119590533-47eae100-be18-11eb-887d-79d706724b11.png)